src/Timer/Timer.js

```diff
@@ -15,7 +15,7 @@ export default function Timer({targetDate, title}) {
 
   const timeLeft = targetDate - date;
 
-  const note = timeLeft > 0 ? "Note: This is assuming that the episode airs on December 25th, 2019 at 20:00 UTC" : "";
+  // const note = timeLeft > 0 ? "Note: This is assuming that the episode airs on December 25th, 2019 at 20:00 UTC" : "";
```

Comment:
It's not a good practice to commit code that is commented out. It would be better to decide what to do with the note. One option is to just remove it. That's only feasible if it's not needed at all. But if it's needed _sometimes_, maybe a better way would be to make it into a feature. I.e. having a boolean prop that controls whether to show the note or not. Yet another option is to accept the whole not text as a prop. More customizable, yay! Even more customizable is to accept a React element instead of a string. And if we use the special prop called `children`, we can allow it to be used like this:

```jsx
<Timer targetDate={someDate}>
  Note text that can also include <strong>jsx</strong>.
</Timer>
```

But it all comes down to deciding what we want. :smile: